src/woofmark.js

```diff
@@ -218,7 +218,21 @@ function woofmark (textarea, options) {
       if (currentMode === 'html') {
         textarea.value = parse('parseHTML', textarea.value).trim();
       } else {
-        textarea.value = parse('parseHTML', editable).trim();
+        var regexp = /\*\*[A-Z][^*]+ \*\*/gi;
```

Comment:
This looks really good! Thanks! Is there a simple Jest test we could write to demo this, and to protect it?

Thank you!!!

Reply:
will do this @jywarren